### ISSUE TITLE
"The form could not be sent" added to search results

### ISSUE THREAD

**Issue by fritzmg** [contao/core-bundle#1194]
When a page contains a form and there are errors in the form, the page title will be set to
```
The form could not be sent - Lorem ipsum dolor
```
See [Form.php#L271](https://github.com/contao/core-bundle/blob/4.4.8/src/Resources/contao/forms/Form.php#L271). However, sometimes this page title is also put into the search index. Meaning the page title for the search result from that page will also say `The form could not be sent - Lorem ipsum dolor`. Shouldn't
```php
$objPage->noSearch = 1;
$objPage->cache = 0;
```
be added [here](https://github.com/contao/core-bundle/blob/4.4.8/src/Resources/contao/forms/Form.php#L268-L272) as well?

**Comment by aschempp** [comment]
I think we should generally not search-index any POST requests?

**Comment by leofeyer** [comment]
They keyword ist submitted via GET.

**Comment by leofeyer** [comment]
We should not. And we probably should not index sub-requests either.

**Comment by leofeyer** [comment]
We should not. Fixed in 7e61acaa274027642c5b2d1492d7562589fa44ef.
